xchange-core/src/main/java/org/knowm/xchange/dto/account/DepositAddress.java

```diff
@@ -0,0 +1,50 @@
+package org.knowm.xchange.dto.account;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public final class DepositAddress implements Serializable {
```

Comment:
I would rename this to `AddressWithTag` as it is used both for deposits and withdrawals

Reply:
Done.